src/components/Placeholder/components/PlaceholderContent/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { ReactElement } from 'react';
+import clsx from 'clsx';
+import Typography from '../../../Typography';
+import { Props } from './types';
+
+
+function PlaceholderContent({
+    className,
+    children,
+}: Props): ReactElement {
+
+
+    const placeholderClasses = clsx(
+        'brew-Placeholder__content',
+        className,
+    );
+
+
+    return (
+        <Typography className={placeholderClasses} as="p" hasBottomMargin variant="body1">
```

Comment:
We can remove the `hasBottomMargin` prop here - it's been removed from the `Typography` component.